src/training_utils.py

```diff
@@ -16,7 +16,7 @@
 from typing import Any, List, Tuple, Union
 
 from src.util_functions import _get_tensor_value
-from src.SMD_cal import calculate_two_point_df, calculate_two_point_3D
+from src.SMD_cal import calculate_two_point_list, list_to_df_two_point, calculate_two_point_3D
```

Comment:
basically what previous function (calculate_two_point_df) was doing is now done by two separate functions (calculate_two_point_list, list_to_df_two_point)